rpmio/rpmio.c

```diff
@@ -797,8 +822,7 @@ static LZFILE *lzopen_internal(const char *mode, int fd, int xz)
 		ret = lzma_easy_encoder(&lzfile->strm, level, LZMA_CHECK_SHA256);
 #ifdef HAVE_LZMA_MT
 	    } else {
-		if (threads == -1)
-		    threads = rpmExpandNumeric("%{getncpus}");
+		threads = get_compression_threads (threads);
```

Comment:
There's an extra space before (threads), both here and in the other get_compression_threads() call.